hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -363,6 +370,13 @@
       name: "USBDEV.POWER.WAKE_BUS_RESET",
       desc: "Wake from Sleep in response to Bus Reset from host."
     }
+    {
+      name: "USBDEV.POWER.TOGGLE_RESTORE",
+      desc: '''Save and restore Data Toggles for all IN and OUT endpoints.
+            Resuming from Deep Sleep requires the ability to restore the Data Toggles
+            to an arbitrary stage. (Requires prod-only RTL changes.)
```

Comment:
Is this a feature that will only appear in prod? (And are we certain that it will be added?) I'm not sure that I understand why we track it here (rather than adding it once the feature is thought to be in the block)

Reply:
It is prod only, yes. The RTL is already built and tested (PR #19269) and it must be included for Deep Sleep to be usable. I raised the question of what to do about a test that will not pass for SV3 and the decision was to annotate it as 'should be SV4 and not expected to pass on ES.'

Reply:
Oh, I see. I guess the head of the master branch describes prod, not really the ES chip (when the two are different). So I'd probably not include the note.

Reply:
The annotation is to avoid wasted effort during SiVal testing; I suspect somebody will at some point wonder why the test/feature is failing on SiVal, since master is being used for both prod and ES presently.

Reply:
Ok, cool that makes sense.